### PR TITLE
Scientific method

### PR DESCRIPTION
**Pull request by eschreiner1717**
Created dual SMA strategy, tuned its hyperparameters, and compared it in sample and out of sample versus other hyperparameter combinations and our OG single SMA strategy.

Performed the scientific method to improve our strategy in an organized fashion.

### CODE REVIEW

**Review by Alon-Barzilay — APPROVED** [review]
I personally like "training data" and "test data" instead of in-sample and out-sample - again personal opinion but I think that makes more sense.

Maybe some more graphs side by side of good strategies vs bad to see what the mannerisms of a good strategy was and how/why the bad was bad.

In the analysis section called "Key Metric Degradation" there is a chunk of questions at the end of it which are of course really good questions but they are not answered. DId you plan on answering these live?

Other than my little comments you killed this! 

Nothing necessarily needs changing so I will approve, but if you would like to continue on this I will happily look at it again!

**Comment by eschreiner1717** [comment]
Thanks Alon you bring up some good points. This notebook is meant to be kinda my "raw" workflow as I was learning and experimenting. I took the time to organize some notes but mainly I'll include the graphs you're talking about in the presentation.

Those questions in Key Degradation were more to generate thinking. There are actually lots of better way to handle the sort of issue I faced with the backtesting periods, but I was curious what people could come up with on their own. There is no best answer.

The terms "training" and "test" data are synonymous to IS and OOS for sure, but I think those are more used in a ML context. I don't wanna change my variable names for the whole notebook, so imma just mention that analogy in the meeting.

**Review by KhuramC — APPROVED** [review]
Alon has worded better than I could, but otherwise, LGTM.
